guestbook/README.md

```diff
@@ -4,704 +4,294 @@
+## Scale the Web Frontend 
+Scaling up or down is easy because your servers are defined as a Service that uses a Deployment controller.
 
-```console
-$ gcloud compute firewall-rules create --allow=tcp:80 --target-tags=kubernetes-node kubernetes-node-80
-```
+1. Run the following command to scale up the number of frontend Pods:
 
-For GCE Kubernetes startup details, see the [Getting started on Google Compute Engine](http://kubernetes.io/docs/getting-started-guides/gce/)
+        kubectl scale deployment frontend --replicas=5
 
-For Google Compute Engine details about limiting traffic to specific sources, see the [Google Compute Engine firewall documentation][gce-firewall-docs].
+{:start="2"}
+2. Query the list of Pods to verify the number of frontend Pods running:
 
-[cloud-console]: https://console.developer.google.com
-[gce-firewall-docs]: https://cloud.google.com/compute/docs/networking#firewalls
+        kubectl get pods
+
+   The response should look similar to this: 
+
+        NAME                            READY     STATUS    RESTARTS   AGE
+        frontend-3823415956-70qj5       1/1       Running   0          5s
+        frontend-3823415956-dsvc5       1/1       Running   0          54m
+        frontend-3823415956-k22zn       1/1       Running   0          54m
+        frontend-3823415956-w9gbt       1/1       Running   0          54m
+        frontend-3823415956-x2pld       1/1       Running   0          5s
+        redis-master-1068406935-3lswp   1/1       Running   0          56m
+        redis-slave-2005841000-fpvqc    1/1       Running   0          55m
+        redis-slave-2005841000-phfv9    1/1       Running   0          55m
+
+3. Run the following command to scale down the number of frontend Pods:
+
+        kubectl scale deployment frontend --replicas=2
+
+4. Query the list of Pods to verify the number of frontend Pods running:
+
+        kubectl get pods
+
+   The response should look similar to this:
+
+        NAME                            READY     STATUS    RESTARTS   AGE
+        frontend-3823415956-k22zn       1/1       Running   0          1h
+        frontend-3823415956-w9gbt       1/1       Running   0          1h
+        redis-master-1068406935-3lswp   1/1       Running   0          1h
+        redis-slave-2005841000-fpvqc    1/1       Running   0          1h
+        redis-slave-2005841000-phfv9    1/1       Running   0          1h
+        
+{% endcapture %}
+
+{% capture cleanup %}
+Deleting the Deployments and Services also deletes any running Pods. Use labels to delete multiple resources with one command.
+
+1. Run the following command to delete all Pods, Deployments, and Services.
+
+        kubectl delete deployments,services -l "app in (redis, guestbook)"
```

Comment:
this is REALLY cool actually TIL you can use comma like that, but it's simultaneously super confusing.

Let's expand this as:

    kubectl delete deployment -l app=redis
    kubectl delete service -l app=redis
    kubectl delete deployment -l app=guestbook
    kubectl delete service -l app=guestbook

the result should be the same.